CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+[42.5.0] (2022-08-23 11:20:11 -0400)
+### Changed
+- fix: revert change in [PR #1986](https://github.com/pgjdbc/pgjdbc/pull/1986) where float was aliased to float4 from float8. 
```

Comment:
Is the "-" prefix for the revert line intentional? Otherwise rest looks fine.

Reply:
Yes, is intentional, when is rendered in markdown it looks like a 
- bullet point